change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -2060,8 +2123,26 @@ PROCEDURE VALIDATECONFIG
        PARAMVALIDATIONERROR=TRUE
        PARAMVALIDATIONERRORCHR="No email detail lines specified for member email."
       END
-
-     ELSE IF PARAMEMAILFROM<>MEMBEREMAIL THEN
+    END
+[* If allow in memo mode is enabled, check for email detail line. If none specified, add default lines.
+*]     
+   IF PARAMALLOWINMEMOMODE=TRUE AND
+      MMEBEMAILLINECOUNT=0 AND
+      MMEBEMAILLINECOUNT+3<EMAILLINESMAX THEN
+    DO [Add default memo email body lines if memo is allowed and configuration lines are blank and fit the array.]
+     PARAMMEMOMODEEMAILTEXT(1)="Nightly processing is underway. Your financial institution address information has been updated"
+     PARAMMEMOMODEEMAILTEXT(2)="and will be reflected the next business day."
```

Comment:
If you're going to potentially have default messages delivered to the member, then they shouldn't be burried in the code but up to in the setup procedure or in an initialization procedure.